File: Array/reverseArrayOrString.java
package Array;

import java.util.Scanner;

class reverseArrayOrString{
    public static void main(String[] args){
        int size=0;
        Scanner sc = new Scanner(System.in);
        size = sc.nextInt();
        int [] arr = new int[size];
        for(int i=0;i<size;i++){
            int value;
            System.out.println("Enter value for "+(i+1)+" element :");
            value = sc.nextInt();
            arr[i] = value;
        }
        for(int i=0;i<arr.length/2;i++){
            int temp = arr[arr.length-1-i];
            arr[arr.length-1-i] = arr[i];
            arr[i] = temp;
        }
        for(int i : arr){
            System.out.print(" "+i);
        }
        sc.close();
        System.out.println("");
    } 
}

File: Array/reverseArrayOrString.cpp
#include<iostream>

using namespace std;

int main(){
    cout << "Enter the size of an array : "<<endl;
    int size;
    cin >>size;
    string arr[size];
    for(int i=0;i<size;i++){
       string value;
        cout<<"Enter value for "<<i+1<<" element : "<<endl;
        cin>>value;
        arr[i]=value;
    }
    for(int j=0;j<size/2;j++){
        string temp = arr[size-j-1];
        arr[size-j-1]=arr[j];
        arr[j] = temp;
    }
    for(int k=0;k<size;k++){
        cout<<" "<<arr[k];
    }
    cout<<endl;
}
